docs/DeveloperGuide.md

```diff
@@ -283,32 +283,49 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the Amigos and the **Actor** is the user, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: E03 - Checking all events**
+
+**Guarantees**
+
+* If any events exist within the system they will be displayed.
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User requests to show all events
+2. Amigos switches the GUI to the events tab.
```

Comment:
Hmm, do you think this step is superfluous? (seems like the intention is the same as step 3.)